core/neat/mixins/_grid-container.scss

```diff
@@ -0,0 +1,35 @@
+/// Creates Neat grid container with clearfix.
+///
+/// @argument {list} $grid
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-container;
+///   }
+///
+/// @example css
+///   .element {
+///     max-width: 1200px;
+///
+///     &::after {
+///       clear: both;
+///       content: "";
+///       display: table;
+///     }
+///   }
+
+@mixin grid-container($grid: $neat-default-grid) {
+  $_max-width: map-get($grid, max-width);
+
+  margin-left: auto;
+  margin-right: auto;
+  max-width: $_max-width;
+
+  &::after {
+    clear: both;
+    content: "";
+    display: table;
```

Comment:
This can be display block, Display table is no longer needed → https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/addons/_clearfix.scss